library/Businessprocess/Renderer/Breadcrumb.php

```diff
@@ -41,7 +42,11 @@ public static function create(Renderer $renderer)
             )
         ));
         $breadcrumb->add(Html::tag('li')->add(
-            Html::tag('a', ['href' => $bpUrl], $bp->getTitle())
+            Html::tag(
+                'a',
+                ['href' => $bpUrl],
+                HtmlElement::create('span', ['class' => 'text no-wrap', 'title' => $bp->getTitle()], $bp->getTitle())
+            )
```

Comment:
For breadcrumbs the text is wrapped with span because `overflow:hidden` will also hide `::before` and `::after` content of anchor tag, that means the arrows of breadcrumbs will be hidden. But it makes sense not to wrap the text in node tiles.